addon/ng2/blueprints/ng2/files/angular-cli-build.js

```diff
@@ -12,8 +12,7 @@ module.exports = function(defaults) {
       'systemjs/dist/system-polyfills.js',
```

Comment:
According to:

https://github.com/systemjs/systemjs

the purpose of system-polyfills is to ensure Promise is available.

Core-js includes a Promise implementation, per:

https://github.com/zloirock/core-js

Therefore, I think this line that copies "system-polyfills.js" should also be removed by this PR.